tests/test_keys.py

```diff
@@ -14,9 +14,7 @@ def test_gen_key():
 
 
 def test_private_key_serialization():
-    pre = umbral.PRE()
```

Comment:
omg right off the bat, very first line - one of the most beautiful redlines I've ever seen.